scripts/empty_upgrade_handler_gen.sh

```diff
@@ -119,3 +120,6 @@ else
     echo "Using pre-defined osmosis-e2e-init-chain tag: $PREV_OSMOSIS_E2E_TAG"
     sed -i '/previousVersionInitTag/s/".*"/'"\"$PREV_OSMOSIS_E2E_TAG\""'/' $e2e_file
 fi
+
+# update OSMOSIS_E2E_UPGRADE_VERSION in launch.json
+sed -i "s/${bracks}OSMOSIS_E2E_UPGRADE_VERSION${bracks}: ${bracks}v$latest_version${bracks}/${bracks}OSMOSIS_E2E_UPGRADE_VERSION${bracks}: ${bracks}$version_create${bracks}/" ./.vscode/launch.json
```

Comment:
nit: what does `bracks` stand for? Can we make this name more descriptive? e.g. `new_version`